cmd/security-secrets-setup/entrypoint.sh

```diff
@@ -24,53 +24,26 @@ set -e
 # thus more graceful termination of all sub-processes if any.
 
 # runtime directory is set per user:
-export XDG_RUNTIME_DIR=${XDG_RUNTIME_DIR:-/run/user/$(echo $(id -u))}
-
-PKI_INIT_RUNTIME_DIR=${XDG_RUNTIME_DIR}/${PKI_INIT_DIR}
+XDG_RUNTIME_DIR=${XDG_RUNTIME_DIR:-/run/user/$(echo $(id -u))}
+PATH="$BASE_DIR:$PATH"
+export XDG_RUNTIME_DIR PATH
 
 # debug output:
 echo XDG_RUNTIME_DIR $XDG_RUNTIME_DIR
-echo PKI_INIT_RUNTIME_DIR $PKI_INIT_RUNTIME_DIR
-
-# configuration for TLS materials:
-PKI_CONFIG_JSON_DIR="res"
-PKI_SETUP_VAULT_FILE=${PKI_CONFIG_JSON_DIR}"/pkisetup-vault.json"
-PKI_SETUP_KONG_FILE=${PKI_CONFIG_JSON_DIR}"/pkisetup-kong.json"
+echo BASE_DIR $BASE_DIR
 
-# check if files exists
-if [ ! -f "${PKI_SETUP_VAULT_FILE}" ]; then
-    echo "Error: certificate config file for Vault is missing"
-    exit 1
+# if running security-secrets-setup subcommand
+# build full command line into positional args
+if [ "$1" = 'generate' -o "$1" = 'cache' -o "$1" = 'import' -o "$1" = 'legacy' ]; then
```

Comment:
Get rid of support for `legacy`?

Reply:
yeap, in another separate PR when we get rid of code for legacy as well.